docs/docs/install/config/index.md

```diff
@@ -89,24 +89,36 @@ A complete config file with all available options can be found [here](https://gi
 
 ### Containers
 
-- To disable container enrichment use: **`--containers enrich=false`**.
+- To enable container enrichment, include the flag: **`--enrichment container.enabled`**. To disable it, simply omit the flag.
```

Comment:
Omitting the flag may not be enough to disable it if setting one of the other `--enrichment container` enables it as syntactic sugar